pricegraph/fuzz/fuzz_targets/orderbook.rs

```diff
@@ -1,9 +1,62 @@
 #![no_main]
+use arbitrary::Arbitrary;
 use libfuzzer_sys::fuzz_target;
-use pricegraph::{Element, Orderbook};
+use pricegraph::{Element, Pricegraph, TokenId, TokenPair};
+use std::iter::once;
+
+// Limit the maximum token id that is allowed to appear in the generated orders. Without this we can
+// make orderbook creation too slow if one order contains a large buy or sell token id.
+const MAX_TOKEN_ID: u16 = 16;
 
 // Fuzz creation and usage of Orderbook.
 
-fuzz_target!(|elements: Vec<Element>| {
-    let _ = Orderbook::from_elements(elements);
+#[derive(Arbitrary, Debug)]
+enum Operation {
+    EstimateExchangeRate {
```

Comment:
There is also `estimate_buy_amount`.

Reply:
Do you mean `order_for_sell_amount`? I haven't included it because it just calls `estimate_exchange_rate` internally.

Reply:
In that case, we should use `order_for_sell_amount` to get more coverage no?